back/app/views/application/_logo_medium_top.mjml

```diff
@@ -1,9 +1,5 @@
 <mj-section padding="20px 25px 0">
   <mj-column>
-    <mj-raw>
-      <%= link_to home_url do %>
-        <%= image_tag logo_url, alt: 'Organization logo' %>
-      <% end %>
-    </mj-raw>
+    <mj-image align="left" width=<%= "#{logo_width}px" %> src=<%= logo_url %> href=<%= home_url %> alt="Organization logo"/>
```

Comment:
Can you click the link? I didn't know the image tag had the `href` attribute.

Also, I think it's totally fine to write `width="80px"` without the helper. In HTML we make assumptions about pixels (see `padding="20px 25px 0"`) and it's fine.

Reply:
Yes can click link.

We set the height of our logos to 80px, width is the retained aspect ratio.

In HTML (and MJML) you can only retain aspect by setting width (and height is auto)... hence the need to know (or calculate the width)

Reply:
It's such a pity we cannot set height without this complicated `logo_width`. It works with CSS `height: 80px;`, but mjml overrides all inline CSS. So, probably it's fine to go as it is.